source/catalog.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Кэт энерджи</title>
+  </head>
+
+  <body>
+    <header class="header">
+      <a class="header__logo" href="">
+        <img class="header__logo-image" src="https://via.placeholder.com/191x39" width="191" height="39"
+          alt="Логотип Кэт энерджи">
+      </a>
+      <nav class="nav">
+        <button class="nav__toggle">Открыть меню</button>
+        <div class="nav__wrapper">
+          <ul class="nav__menu site-menu">
+            <li class="site-menu__item">
+              <a href="#">Главная</a>
+            </li>
+            <li class="site-menu__item site-menu__item--active">
+              <a>Каталог продукции</a>
+            </li>
+            <li class="site-menu__item">
+              <a href="#">Подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <h1 class="main_title">Каталог продукции</h1>
+      <section class="products">
+        <div class="products__wrapper">
+          <ul class="products__list">
+            <li class="products__item">
```

Comment:
```suggestion
            <li class="products__item product">
```
Так же, в продукте у тебя странный порядок еллементов идет. 


![image](https://user-images.githubusercontent.com/22337601/77853107-0b5b2f80-71eb-11ea-852a-05178793e89e.png)

product__image
product__info
product__action

Так будет правильный порядок еллементов, и тебе будет их проще позиционировать потом с помощью flexbox.

Грубо говоря, так мы сделаем отдельный БЕМ блок product, который потом сможем реюзать.